pkg/networkmanager/interfaces.js

```diff
@@ -1331,7 +1331,9 @@ PageNetworking.prototype = {
         function highlight_netdev_row(event, id) {
             $('#networking-interfaces tr').removeClass('highlight');
             if (id) {
-                $('#networking-interfaces tr[data-interface="' + shell.esc(id) + '"]').addClass('highlight');
+                $('#networking-interfaces tr')
+                    .attr("data-interface", id)
+                    .addClass('highlight');
```

Comment:
The new code sets all `data-interface` attributes of all rows to `id`, no?  I agree we should stop constructing a jQuery expression here.  I think we need to explicitly loop like this

```
$('#network-interfaces tr').each(function (i, tr) {
  if ($(tr).attr('data-interface') == id)
    $(tr).addClass('highlight');
});
```